lib/index.js

```diff
@@ -12,7 +12,12 @@ var ParserInline = require('./parser_inline');
 var LinkifyIt    = require('linkify-it');
 var mdurl        = require('mdurl');
 var punycode     = require('punycode');
+var trimRight    = require('trim-right');
 
+if (!String.prototype.trimRight) {
```

Comment:
I'd like to avoid microdependencies and prototype changes. That's too complex to just check that source string length > 0.